frontend/src/components/AddItem.tsx

```diff
@@ -57,30 +57,33 @@ export default function AddItem(props: AddItemProps) {
 
     return (
         <div>
-            <Button onClick={handleOpen}>Zutat hizufügen</Button>
+            <Button onClick={handleOpen}>Zutat eintragen</Button>
             <Dialog open={open} onClose={handleClose}>
                 <DialogTitle>Neue Zutat</DialogTitle>
                 <DialogContent>
                     <TextField
                         label="Name"
                         name={"name"}
+                        placeholder={"Zutatname"}
```

Comment:
I don´t know if "Zutatname" is right written. Please check.

Reply:
I made it short :) now it is only Zutat